utils/experiment_options/experimentoptions.py

```diff
@@ -204,13 +183,15 @@ def __repr__(self):
                     \n    geo_long = {} degrees\
                     \n    altitude = {} metres \
                     \n    boresight = {} degrees from geographic north, CCW = negative. \
+                    \n    boresight_shift = {} degrees. \
                     \n    beam_sep = {} degrees\
                     \n    velocity_sign = {} \
-                    \n    analog_rx_attenuator = {} dB \
-                    \n    tdiff = {} us \
+                    \n    tdiff_a = {} us \
```

Comment:
```suggestion
                    \n    tdiff_a = {} ns \
```
It is ns everywhere else, assuming this was a typo